src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -46,23 +59,111 @@ public ConsoleLoggerProvider(IOptionsMonitor<ConsoleLoggerOptions> options)
             }
         }
 
+        private static bool DoesConsoleSupportAnsi()
+        {
+#if TargetsWindows
```

Comment:
It seems like you're trying to write os specific code however this package is not cross compiled for OS it has:

```xml
<TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0</TargetFrameworks>
```

We would need to instead do:

```xml
<TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;netstandard2.0-Windows_NT</TargetFrameworks>
```

Also, we don't define a compile constant `TargetsWindows` so even if we cross compile, this code will always just include the `#else` block. 

I would suggest adding a `ConsoleLoggerProvider.AnyOS.cs` that does:

```cs
private static bool DoesConsoleSupportAnsi() => true;
```

and a `ConsoleLoggerProvider.Windows.cs with the windows specific code. You will need to make this class partial to do that.

cc: @ericstj